services/backend/src/resources/user/service/models/model_user.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+type UserModel struct {
+	ID            string
+	Username      string
+	DisplayName   string
+	Email         string
+	EmailVerified bool
+}
+
+func NewUserModel(id, username, displayName, email string, emailVerified bool) *UserModel {
+	return &UserModel{
+		ID:            id,
+		Username:      username,
+		DisplayName:   displayName,
+		Email:         email,
+		EmailVerified: emailVerified,
+	}
+}
```

Comment:
It gets used by the storage layer. I cannot simply convert it, because if I import the storage layer, I would get an import loop. Also, I just like constructor functions. Just my opinion.

Reply:
and validation is also not needed because it is again a service layer model, so just transportation, no user input. So no worries